doc/config.toml

```diff
@@ -0,0 +1,24 @@
+baseURL = "http://example.org/3.0.0"
+languageCode = "en-us"
+title = "LoRaWAN Stack"
```

Comment:
The Things Network Stack for LoRaWAN V3

Reply:
Feels wrong to put the V3 after LoRaWAN, make it sound like we are supporting LoRaWAN version 3. Will add a `-`

Reply:
I'd say "The Things Network Stack for LoRaWAN" is sufficient. We'll have the version somewere else on the page, right?